app/workers/rapid_ready_for_decision/form526_asthma_job.rb

```diff
@@ -32,7 +32,17 @@ def query_and_assess_lighthouse(form526_submission)
     def assess_medications(medications)
       return [] if medications.blank?
 
-      RapidReadyForDecision::LighthouseMedicationRequestData.new(medications).transform
+      asthma_keywords = RapidReadyForDecision::Constants::DISABILITIES[:asthma][:keywords]
+      transformed_medications = RapidReadyForDecision::LighthouseMedicationRequestData.new(medications).transform
+      flagged_medications = transformed_medications.map do |medication|
+        {
+          **medication,
+          flagged: asthma_keywords.any? do |keyword|
+            medication.to_s.downcase.include?(keyword)
+          end
+        }
+      end
+      flagged_medications.sort_by { |medication| medication[:flagged] ? 0 : 1 }
```

Comment:
This looks like it's working well! I wonder if we'd want to do any additional sorting (by name or prescription date) but that's not in the design, so just a side comment.

Reply:
@leikkisa so there is additional generic sorting done in the medication data request class. Because this sorting is specific to asthma I put it here. Lemme know if you have any more concerns downstream...